packages/opentelemetry-resources/src/platform/browser/ServiceInstanceIDDetector.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { noopDetector } from '../../detectors/NoopDetector';
+
+export const serviceInstanceIDDetector = noopDetector;
```

Comment:
We can also drop this any only use the sync one. :slightly_smiling_face: 
(see longer comment on the Node.js implementation of it)

Reply:
Done